ds701_book/jupyter_notebooks/11-Dimensionality-Reduction-SVD-II.ipynb

```diff
@@ -5,259 +5,259 @@
       "metadata": {},
```

Comment:
<p>Perhaps add some summary remarks about what we see from the plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>